2.Lets-Build-OTPublisher/Lets-Build-OTPublisher/ViewController.m

```diff
@@ -74,7 +74,7 @@ - (BOOL)shouldAutorotateToInterfaceOrientation:
  */
 - (void)doConnect
 {
-    OTError *error;
+    OTError *error = nil;
```

Comment:
Theoretically yes, but when I ran this code it caused a crash. Adding the nil fixed that.